heartbeat/CTDB.in

```diff
@@ -580,27 +751,24 @@ ctdb_start() {
 		return $OCF_ERR_GENERIC
 	fi
 
+	local version=$(ctdb_version)
 	# Generate new CTDB sysconfig
 	generate_ctdb_sysconfig
-	enable_event_scripts
```

Comment:
generate_ctdb_sysconfig() should be called only for pre 4.9.0 branches.

Reply:
@amitay does /etc/sysconfig/ctdb have any effect on 4.9.0 ? I.e. should we be removing (the autogenerated config) in 4.9+ it if it exists?

Reply:
CTDB does not read /etc/sysconfig/ctdb any more except one case.  In case you have a system without systemd and using rc scripts to start ctdb.  But I guess you are not concerned with that case since you start ctdb on your own.

You can definitely remove (or rename) that file if it exists from previous installations.